arches_querysets/datatypes/resource_types.py

```diff
@@ -14,51 +13,51 @@
 
 class ResourceInstanceDataType(datatypes.ResourceInstanceDataType):
     def transform_value_for_tile(self, value, **kwargs):
+        parsed = super().transform_value_for_tile(value, **kwargs)
+        if parsed is None:
+            parsed = value
```

Comment:
The super() method doesn't handle incoming values like `models.ResourceInstance`, that is, it implicitly returns None, so I just fall back to what was passed in.